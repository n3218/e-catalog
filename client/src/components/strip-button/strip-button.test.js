import { shallow, mount, render } from "enzyme"
import React from "react"
import StripeCheckoutButton from "./strip-button.component"

describe("CheckoutItem tests", () => {
  let mockPrice
  let wrap

  beforeEach(() => {
    mockPrice = "20"
    wrap = shallow(<StripeCheckoutButton price={mockPrice} />)
  })

  it("expect to render StripeCheckoutButton component", () => {
    expect(wrap).toMatchSnapshot()
  })
})
