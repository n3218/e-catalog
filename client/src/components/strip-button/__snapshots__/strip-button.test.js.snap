// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutItem tests expect to render StripeCheckoutButton component 1`] = `ShallowWrapper {}`;
